src/jit/importer.cpp

```diff
@@ -14819,6 +14819,11 @@ void Compiler::impImportBlockCode(BasicBlock* block)
                 }
                 op1->gtFlags |= GTF_EXCEPT;
 
+                if (prefixFlags & PREFIX_UNALIGNED)
+                {
+                    op1->gtFlags |= GTF_IND_UNALIGNED;
+                }
+
```

Comment:
Technically, I think this isn't valid to push down, since gtNewObjNode returns an OBJ, meaning you need to use GTF_BLK_UNALIGNED. (Unfortunately, we have no dynamic checking, and in fact that is the same value as GTF_IND_UNALIGNED right now, but you still should use the "right" one).